boards/arm/mps3_an547/mps3_an547.dts

```diff
@@ -85,6 +86,21 @@
 		};
 	};
 
+	ethosu {
```

Comment:
Question: shouldn’t this be defined at soc dts files (not board)?

Reply:
Possibly. But I think this is OK for something as new as this MPU. There's also no other boards with that SoC.

Reply:
There is no SoC .dtsi for this SoC, so I'd rather leave it here to be honest.

Reply:
Yeah, we seem to have done this inlining of the "soc" into the board for all the MPS2 AN style boards.

Reply:
I guess the assumption was that the 'SoC' would never exist outside a single combination of FPGA board + ANxxx bitstream. This particular board family is definitely a bit of a weird niche solution that found a second life solving specific CI problems.